public/facesmashmain.css

```diff
@@ -0,0 +1,105 @@
+*{
+	box-sizing: border-box;
+	font-size:24px;
+}
+
+.clearfix{
+	overflow: auto;
+}
+
+body {
+	background-color: #EDEDED;
+
+}
+
+.fixedheader {
+	background-color: #3B5998;
+	color: #888888;
+	margin: 0;
+	padding: 0;
+
+
```

Comment:
it's good to remove any empty lines between properties and the curly braces. In this case, I'd delete 20-21 and any more in the file that are inside of a css rule.